WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -142,9 +142,9 @@ await foreach (var offendingAlices in CheckTxoSpendStatusAsync(round, cancel).Co
 					{
 						// This should never happen.
 
-						Logger.LogError($"{nameof(CoinVerifier)} has failed to verify all Alices({round.Alices.Count}).", exc);
+						Logger.LogError($"{nameof(CoinVerifier)} has failed to verify all Alices({round.Alices.Count}) for Round({round.Id}).", exc);
 						CoinVerifier.VerifierAuditArchiver.LogException(round.Id, exc);
-						round.EndRound(EndRoundState.AbortedWithError);
+						throw;
```

Comment:
This whole try-catch block can be removed since its wrapped in a try-catch block. 
We could move both logging call there, too.

If that would make sense, pelase tell and I will make the changes.

Reply:
Logger.LogError can be removed